src/groups/mqb/mqbblp/mqbblp_cluster.h

```diff
@@ -422,11 +423,10 @@ class Cluster : public mqbi::Cluster,
 
     void onRelayPushEvent(const mqbi::DispatcherEvent& event);
 
-    bool validateMessage(mqbi::QueueHandle**       queueHandle,
-                         bsl::ostream*             errorStream,
```

Comment:
`errorStream` is removed, but mentioned in the docs below

Reply:
Also, method returns not the bool now but the `ValidationResult` enum, so it should be in docs